ops-bedrock/l2-erigon.sh

```diff
@@ -15,9 +14,6 @@ COMMON_FLAGS=" \
 
 ERIGON_FLAGS=" \
   ${COMMON_FLAGS} \
-  --mine \
```

Comment:
Removing the etherbase and the sigfile seems fine, but if `--mine` isn't enabled, then I don't believe erigon will build blocks and act as a sequencer.

I just tried removing it in `op-e2e/extrernal_erigon/main.go` and the tests simply hang for me.  If we're removing it here, I'd think we should remove it there as well?

Reply:
I have removed it in our sequencer node. It's still building new blocks.

Reply:
Looks like this breaks the e2e in CI as well -- does it work on your system?

Reply:
No, I am trying to figure out why. The integration tests work, but op-e2e tests don't.

Reply:
Ok, figured out. The `Regeneration ended` observation should be removed. Geth also has this `--mine` option and it is not enabled in the op-e2e tests. I think we should be good to remove it.